packages/store-cache/src/utils.ts

```diff
@@ -257,7 +260,7 @@ function tupleToRecord(tuple: Tuple): Record<string, any> {
  * (see https://github.com/ccorcos/tuple-database/issues/25)
  * For now only `bigint` needs serialization.
  */
-function serializeKey(key: unknown) {
+function serializeKey(key: string | number | bigint | boolean): string | number | boolean {
```

Comment:
The casting above was mostly so I could make these utils more specific in terms of types. Happy to roll this back if you prefer how it was.